linux_os/guide/system/logging/log_rotation/ensure_logrotate_activated/rule.yml

```diff
@@ -35,6 +35,7 @@ references:
     cobit5: APO11.04,BAI03.05,DSS05.04,DSS05.07,MEA02.01
     iso27001-2013: A.12.4.1,A.12.4.2,A.12.4.3,A.12.4.4,A.12.7.1
     cis-csc: 1,14,15,16,3,5,6
+    cis@rhel8: 4.3
```

Comment:
The 4.3 needs to be quoted because it's casted to float implicitly, which causes build error.

Reply:
@jan-cerny how can this be fixed? References are always going to be strings.

Reply:
added